erpnext/setup/workspace/erpnext_settings/erpnext_settings.json

```
@@ -27,6 +27,14 @@
    "link_to": "Projects Settings",
    "type": "DocType"
   },
+  {
+   "color": "Grey",
+   "doc_view": "",
+   "icon": "dot-horizontal",
+   "label": "Naming Series",
+   "link_to": "Document Naming Rule",
```

```suggestion
   "link_to": "Naming Series",
```

@sumaiya2908  I am not sure. Is this supposed to be linked to naming series (page/tool) or list of naming series (document naming rule)?